files/messages/welcome.txt

```diff
@@ -1,20 +1,34 @@
-Hello everyone, and thank you for signing up for our upcoming instructor training session. In order to prepare:
+Hello everyone,
+
+Thank you for signing up for our upcoming instructor training workshop. 
+
+Please this website for specific details about the workshop [INSERT WORKSHOP WEBSITE LINK HERE].
+
+Importantly, class will be from 9:00 am - 4:15 pm both days. We will have a one hour lunch break and two short coffee breaks each day. [INSERT WHETHER OR NOT FOOD AND DRINKS ARE SUPPLIED BY HOST].
+
+**In order to prepare:**
 
 1.  Please read these two short papers, which provide a brief overview of some key evidence-based results in teaching:
 
     * "The Science of Learning" (https://swcarpentry.github.io/instructor-training/files/papers/science-of-learning-2015.pdf)
     * "Success in Introductory Programming: What Works?" (https://swcarpentry.github.io/instructor-training/files/papers/porter-what-works-2013.pdf)
 
-2.  Please go to the Software Carpentry lessons page (http://software-carpentry.org/lessons/) and the Data Carpentry lessons page (http://www.datacarpentry.org/lessons/), have a look at what we currently teach, and then choose *one* episode from the list at the bottom of this message and read through it carefully. You will be using your selected episode for several in-class exercises, so be sure you are comfortable with the content.
+2.  Please go to the Software Carpentry lessons page (http://software-carpentry.org/lessons/) and the Data Carpentry lessons page (http://www.datacarpentry.org/lessons/), have a look at what we currently teach, and then choose *one episode* from the list at the bottom of this message and read through it carefully. You will be using your selected episode for several in-class exercises, so be sure you are comfortable with the content.
+
+**What to bring to the workshop:**
+
+1.  We will be recording one another teaching in pairs or threes during the class, so please bring a cell phone or hand-held camera that can record audio and video. It doesn't have to be high-quality, but it should be good enough that you can understand what someone is saying.
+
+2.  Please also bring a laptop powerful enough to teach on so that you can take part in all of the practical exercises (i.e. SQL, Git, Python, R, or Unix) and a power cable. You will need this laptop on both day of the instructor training.
```

Comment:
This makes it sound like we will do a lot of exercises in all these languages. Perhaps drop the list of languages and rephrase to make clear that participant is expected to teach something from his chosen lesson?